package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "demux-postgres",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
I think demux-eos is on a 1.0.x version, do we want to bump this to a 1.0.x as well?

Reply:
I gave this some thought, and I realized that while breaking changes would necessitate a version increase of descendent class packages, that does not preclude the possibility of a descendent class package from having a breaking change that does not coincide with a major demux release. Therefore I think if we attempt to coincide them artificially, then we'll be setting a bad precedent- we should either keep them definitely synced or not.